frontend/app/views/event/page.scala.html

```diff
@@ -49,7 +49,12 @@ <h1 class="event-masthead__name qa-event-detail-name">@event.name.text</h1>
                     }
                 </div>
             }
-            <img class="event-header__logo" src="@Asset.at("images/providers/" + event.providerOpt.getOrElse(event.metadata.identifier) + ".svg")">
+            @if(event.providerOpt.getOrElse(event.metadata.identifier) == "5x15") {
```

Comment:
That was where I was going, but wasn't sure how to turn this line:

    if EBEvent.providerWhitelist.contains(provider)

... into something that would work with a tuple/class.